### PR TITLE
scheme/tpm-enacttrust: update  node ID handling

### PR DESCRIPTION
**Pull request by setrofim**
A recent changed in the EnactTrust attestation token format  means that the attesting node ID is now in the first 16 bytes of the token, and is not embedded inside the TPMS_ATTEST structure.

### CODE REVIEW

**Review by thomas-fossati — APPROVED** [review]
LGTM

File: scheme/tpm-enacttrust/test/cmd/gen-token/README.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+An attestation token has the following format:
+
+    TPMS_ATTEST_LEN||TPMS_ATTEST||TPMS_ATTEST_SIGNATURE
+
+where
+
+`TPMS_ATTEST_LEN` is a uint16 values containing the length (in bytes) of the
+following `TPMS_ATTEST` structure. `TPMS_ATTEST` is structured according to
+[TPM 2.0
+Specification](https://trustedcomputinggroup.org/wp-content/uploads/TCG_TPM2_r1p59_Part2_Structures_pub.pdf).
```
Comment (yogeshbdeshpande) [inline scheme/tpm-enacttrust/test/cmd/gen-token/README.md:17]: .nit: Is this new line intentional

Comment (setrofim) [reply scheme/tpm-enacttrust/test/cmd/gen-token/README.md:17]: yes, this has been automatically formatted to adhere to 79 columns (the standard to README files).

File: scheme/tpm-enacttrust/test/cmd/gen-token/README.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+{
+  "node-id": "7df7714e-aa04-4638-bcbf-434b1dd720f1",
+  "firmware": 7,
+  "pcrs": [1, 2, 3, 4],
+  "algorithm": 4,
+  "digest": "h0KPxSKAPTEGXnvOPPA/5HUJZjHl4Hu9eg/eYMTPJcc="
+}
+```
+
+note: "signer" and "digest" are decoded as `[]byte`, and so, as per
```
Comment (yogeshbdeshpande) [inline scheme/tpm-enacttrust/test/cmd/gen-token/README.md:35]: Just a clarification question: Where is the "signer" part?

**Review by thomas-fossati — APPROVED** [review]
re-🚢 it!

:-)

File: scheme/tpm-enacttrust/test/cmd/gen-token/README.md (hunk 1 above)
Comment (yogeshbdeshpande) [reply scheme/tpm-enacttrust/test/cmd/gen-token/README.md:17]: Thanks!

File: scheme/tpm-enacttrust/test/cmd/gen-token/README.md (hunk 2 above)
Comment (setrofim) [reply scheme/tpm-enacttrust/test/cmd/gen-token/README.md:35]: erm good point. No idea. I'm guessing it was there at some point, but then was unecessary. Will remove

**Review by yogeshbdeshpande — APPROVED** [review]

**Pull request merged**